apps/prairielearn/src/pages/instructorStudents/instructorStudents.html.tsx

```diff
@@ -277,8 +277,7 @@ function StudentsCard({
         cell: (info) => <EnrollmentStatusIcon type="text" status={info.getValue()} />,
         filterFn: (row, columnId, filterValues: string[]) => {
           if (filterValues.length === 0) return true;
-          const current = row.getValue(columnId);
-          if (typeof current !== 'string') return false;
+          const current = row.getValue<StudentRow['enrollment']['status']>(columnId);
```

Comment:
Some cleanup I noticed. I looked into the types for `getValue` for filters, but they don't have enough info. I can dig up the issue if needed, this is a type bug in tanstack table.

Reply:
Fixed.